### DISCUSSION TITLE
太费劲，太难用

### DISCUSSION THREAD

**Discussion by cyjeen** [community/community#65059] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

太费劲了，搞了整整两天都找不到RxHttp，某个时候不小心生成了RxHttp，但是也没有addClass, addQuery, toObservable等这些东西。今天按流程升级到3.1.1，按github步骤，每个步聚都不敢省，就怕生成不了RxHttp，可是很遗憾，最后还是没有生成RxHttp，而且怎么搞都没有，这个东西就不是让人拿来用的，就是用来折磨人的，还不如直接上okhttp，太费劲，毫无实用性，比供个祖宗都更难伺候。

**Comment by queenofcorgis** [comment]
Hi @cyjeen , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
